### PR TITLE
Fix release for auto-discovery

### PR DESCRIPTION
**Pull request by Fruity-Grebbles**
Adds files needed for discovery and changes names, titles, and paths

### CODE REVIEW

File: exts/siborg.human.generator/config/extension.toml [hunk 1]
```diff
@@ -34,17 +34,13 @@ keywords = ["kit", "makehuman","human","character"]
 "omni.kit.browser.core" = {}
 "omni.kit.browser.folder.core" = {}
 
-
 # Main python module this extension provides, it will be publicly available as "import omni.hello.world".
 [[python.module]]
-name = "siborg.create.human"
+name = "siborg.human.generator"
```
Comment (cadop) [inline exts/siborg.human.generator/config/extension.toml:39]: please keep siborg.create.human

**Review by cadop — CHANGES_REQUESTED** [review]

File: exts/siborg.human.generator/config/extension.toml (hunk 1 above)
Comment (Fruity-Grebbles) [reply exts/siborg.human.generator/config/extension.toml:39]: Which should be which?

Comment (Fruity-Grebbles) [reply exts/siborg.human.generator/config/extension.toml]: Restored

Comment (Fruity-Grebbles) [reply exts/siborg.human.generator/config/extension.toml]: Needs to be restored elsewhere

File: exts/siborg.create.human/siborg/create/human/browser/model.py [hunk 2]
```diff
@@ -2,16 +2,16 @@
 from typing import List, Union
 import carb.settings
 import omni.kit.commands
-from siborg.create.human.mhcaller import MHCaller
-from siborg.create.human.ui_widgets import DropList
+from siborg.human.generator.mhcaller import MHCaller
```
Comment (cadop) [inline exts/siborg.create.human/siborg/create/human/browser/model.py:5]: still old

**Review by cadop — CHANGES_REQUESTED** [review]

File: exts/siborg.create.human/siborg/create/human/browser/model.py (hunk 2 above)
Comment (Fruity-Grebbles) [reply exts/siborg.create.human/siborg/create/human/browser/model.py]: Fixed

**Review by cadop — APPROVED** [review]

**Pull request merged**